frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Topology smoke test
+	As a user, I want to verify topology feature    
+
+@regression, @smoke
+Scenario: Empty state of topology
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user creates a new project
+   Then user sees Add page with message on the top "No workloads found"
```

Comment:
The then statement should also have the cards on the +Add page.

Reply:
Please add one more Then statement

Reply:
But with verifying empty view with respect to the already present test cases only "No workloads found" is used to verify as it is characteristic feature relating to empty topology view